readme.md

```diff
@@ -16,6 +16,23 @@ console.log(safeStringify(o))  // '{"a":1,"o":"[Circular]"}'
 console.log(JSON.stringify(o)) //<-- throws
 ```
 
+### toJSON support
+
+`fast-safe-stringify` would not attemp to detect circular dependencies
```

Comment:
This is not correct, as the current implementation will still _not do_ that. The major difference is that it will attempt to `decirc` the object upon which `toJSON` was attached to (that is why I did use a `Symbol` on the object), not on the return value of `toJSON`.